packages/x-date-pickers-pro/src/DateRangePicker/DateRangePickerView.tsx

```diff
@@ -13,11 +13,8 @@ import {
   BaseDateValidationProps,
   DayValidationProps,
 } from '@mui/x-date-pickers/internals';
-import {
-  DateRange,
-  CurrentlySelectingRangeEndProps,
-  DayRangeValidationProps,
-} from '../internal/models/dateRange';
+import { DateRange, CurrentlySelectingRangeEndProps } from '../internal/models/range';
```

Comment:
Off topic: I guess the `DateRange` name is becoming confusing
But that's not something easy to change given that it is used everywhere, including by users :/

Reply:
Yes that's why I kept it, and used the import path to indicate that it's not only for date